bq2dbt/bq2dbt.py

```diff
@@ -146,15 +152,15 @@ def bq2dbt():
         `{project_id}.{dataset_id}.{table_name}`  -- Don't leave this in your DBT Statement
     """
 
-    output_path = f"./output/{project_id}/{dataset_id}"
+    output_path = f"./{output_folder}/{project_id}/{dataset_id}"
     os.makedirs(output_path, exist_ok=True)
 
-    logger.info(f"Generating YAML and SQL files to path: {output_path}")
+    logger.info("Generating YAML and SQL files to path: %s", output_path)
```

Comment:
Why did you switch from fstring here ?

Reply:
I follow this pylint tip : https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
I read it and quite wondered why this is a recommendation of pylint. My guess is that %s formatting is less permissive and it's important in the case of logging.

That's too bad because I like fstrings and I try to be consistent in the use of interpolation method...

Anyway, let's go with the % interpolation.